build.sbt

```diff
@@ -80,7 +80,7 @@ lazy val zhttp = (project in file("./zio-http"))
           new URL("https://github.com/amitksingh1490"),
         ),
       ),
-    libraryDependencies ++= Seq(`zio`, `zio-streams`, netty, `scala-compact-collection`, `netty-incubator`),
+    libraryDependencies ++= Seq(`zio`, `zio-logging`, `zio-streams`, netty, `scala-compact-collection`, `netty-incubator`),
```

Comment:
I think we shouldn't have a hard dependency on zio-logging.

Reply:
how we can achieve this ? should we have a subproject zio-http-logging with the LogMiddleware and the zio-logging dependency isolated
or is there any other way to solve it? (something like a conditional dep or feature, I'm a little rusty with scala/sbt)

Reply:
Logging is becoming a part of zio core with 2.0. So is this hard split necessary?